ui/admin/tests/acceptance/credential-store/credentials/update-test.js

```diff
@@ -128,57 +218,118 @@ module(
       );
     });
 
-    test('can discard unsaved credential changes via dialog', async function (assert) {
+    test('can discard unsaved username & password credential changes via dialog', async function (assert) {
       assert.expect(5);
       const confirmService = this.owner.lookup('service:confirm');
       confirmService.enabled = true;
-      assert.notEqual(instances.credential.name, 'random string');
-      await visit(urls.credential);
+      assert.notEqual(instances.usernamePasswordCredential.name, MOCK_INPUT);
+      await visit(urls.usernamePasswordCredential);
       await click('form [type="button"]', 'Activate edit mode');
-      await fillIn('[name="name"]', 'random string');
-      assert.strictEqual(currentURL(), urls.credential);
+      await fillIn('[name="name"]', MOCK_INPUT);
+      assert.strictEqual(currentURL(), urls.usernamePasswordCredential);
       try {
         await visit(urls.credentials);
       } catch (e) {
-        assert.ok(find('.rose-dialog'));
+        assert.dom('.rose-dialog').exists();
```

Comment:
@cameronperera @ZedLi thoughts on this change ?

Reply:
I personally use `isVisible()` for when checking an actual user should be clicking on something

Reply:
ooh okay that's a good rule to follow!